lib/search.js

```diff
@@ -17,28 +17,47 @@ module.exports = new class LibSearch {
     }
 
     optimizeQueryTerms(input) {
+        // parse
+        let normalize = this.normalizeProt();
+        input.imdbid = input.imdbid && parseInt(input.imdbid.toString().replace('tt', ''), 10)
+        input.season = input.season && parseInt(input.season)

```

Comment:
you're setting a field and value assuming there is an input.season, which is not true for movies